runtime/libia2/include/ia2.h

```diff
@@ -174,6 +174,13 @@ uint32_t ia2_get_pkru();
 /// Returns the current compartment pkey
 size_t ia2_get_pkey();
 
+/// lib must be a single DSO or "main" for the executable
+/// extra_libraries must be a semicolon-separated list of libraries
```

Comment:
Where are these coming from? Could they be an array of pointers to strings instead?